experimenter/experimenter/nimbus-ui/src/components/PageSummary/index.tsx

```diff
@@ -373,3 +381,27 @@ const StatusPill = ({
     {label}
   </Badge>
 );
+
+const AlertWarning = ({
+  text,
+  testId,
+  learnMoreLink,
+  learnMoreText = "Learn more",
+  variant = "danger",
+}: {
+  text: string | SerializerMessage;
+  testId: string;
+  learnMoreLink?: string;
+  learnMoreText?: string;
+  variant?: string;
+}) => (
+  <Alert data-testid={`warning-${testId}`} variant={variant}>
+    {text}
+    {learnMoreLink && (
```

Comment:
Include a space so the learn more link is separated from the error text. We don't do this currently, but we should.

```suggestion
    {text}
    {" "}
    {learnMoreLink && (
```

Reply:
done! <img width="1320" alt="image" src="https://github.com/mozilla/experimenter/assets/43795363/52b0828c-f838-4daa-a8b9-f39fc0a51f29">